src/pages/lessons/[slug].tsx

```diff
@@ -567,11 +568,11 @@ const Lesson: React.FC<LessonProps> = ({
           >
             <div className={cx({hidden: !playerVisible})}>
               <Player
-                poster={lesson.thumb_url}
                 canAddNotes={isEmpty(viewer) ? false : !isFullscreen}
                 className="font-sans"
                 container={fullscreenWrapperRef.current || undefined}
                 controls={<DownloadControl lesson={lesson} />}
+                // poster={lesson.thumb_url}
```

Comment:
if the thumbnail image was the first frame of the video then it would work, but this felt clunky.

Reply:
what would be cool is an og image style generator that gave a stylish consistent poster frame